### PR TITLE
Migrate asset file provabrasil.css

### PR DESCRIPTION
**Pull request by raulbcs**
This PR closes issue [QH-100](https://portalqedu.atlassian.net/browse/QHR-100).
 
## Main changes
 
- Migrate asset file `provabrasil.css`

### CODE REVIEW

File: src/GimmeBundle/Resources/assets/gulpfile.js [hunk 1]
```diff
@@ -12,7 +12,8 @@ var jsCmdVersionOne  = uglifyjsBinary + ' <%= file.path %> --define __DEV__=1 --
     jsCompileCoffee  = coffeeBinary   + ' --print --stdio < <%= file.path %>',
     jsUglifyQuotes   = uglifyjsBinary + ' <%= file.path %> --beautify beautify=false,quote_keys=true --no-copyright',
     cssCmdVersionOne = lessBinary     + ' <%= file.path %> --compress',
-    cssCmdVersionTwo = lessBinary     + ' <%= file.path %> --compress' + ' | ' + uglifycssBinary;
+    cssCmdVersionTwo = lessBinary     + ' <%= file.path %> --compress' + ' | ' + uglifycssBinary,
+    cssCmdInputPath  = lessBinary     + ' -x --include-path=generic/:stylesheet/src/ <%= file.path %> ';
```
Comment (marcusesa) [inline src/GimmeBundle/Resources/assets/gulpfile.js]: Unused variable.

File: src/GimmeBundle/Resources/assets/gulpfile.js [hunk 2]
```diff
@@ -118,4 +119,168 @@ gulp.task('provabrasil.js', function() {
     .pipe(gulp.dest('./javascript/dist/'))
 });
 
-gulp.task('default', ['header.js', 'landingideb.js', 'banner.css', 'landingideb.css', 'dropdown-select2.css', 'provabrasil.js']);
+gulp.task('provabrasil-concat.css', function() {
```
Comment (marcusesa) [inline src/GimmeBundle/Resources/assets/gulpfile.js]: Tasks name `provabrasil.css` is better.

File: src/GimmeBundle/Resources/assets/gulpfile.js [hunk 3]
```diff
@@ -118,4 +119,168 @@ gulp.task('provabrasil.js', function() {
...
+        './stylesheet/src/Meritt/QEdu/UI/Footer/InnerFooter.less',
+        './stylesheet/src/Meritt/QEdu/UI/Footer/InnerIdebModal.less',
+        './stylesheet/src/Meritt/QEdu/UI/Follow/Button.less',
+        './stylesheet/src/Meritt/QEdu/UI/Follow/Count.less',
+        './stylesheet/src/Meritt/QEdu/UI/Modal/FillMoreInfoIfSecretaryModal.less',
+        './stylesheet/src/Meritt/QEdu/UI/Modal/FillProfileModal.less',
+        './stylesheet/src/Meritt/QEdu/UI/Modal/InnerModal.less',
+        './stylesheet/src/Meritt/QEdu/UI/Follow/InnerFollowCampaignModal.less'
+    ])
+        .pipe(concat('pb-concat.css'))
```
Comment (marcusesa) [inline src/GimmeBundle/Resources/assets/gulpfile.js]: Check indentation.

**Comment by raulbcs** [comment]
Removed unused variable and identation fixed @ 6bae046

**Review by marcusesa — APPROVED** [review]

**Pull request merged**